avae/utils.py

```diff
@@ -228,3 +230,29 @@ def colour_per_class(classes: list):
     # Generate a list of colors based on the modulo operation of i with respect to the number of colors in the combined colormap
     colours = [custom_cmap(i % len(combined_cmap)) for i in range(num_colors)]
     return colours
+
+
+def pose_interpolation(
```

Comment:
Can we add some documentation to this function? As is a complex one and has many different variables.

Reply:
This is done now